src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -78,6 +78,11 @@ endif()
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/..")
 target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${OFFLINE_TRANSFORMATIONS_LIB})
 
+if(DEFINED CI_BUILD_NUMBER)
+    message(STATUS "Python CI Build Number: ${CI_BUILD_NUMBER}")
+    target_compile_definitions(${PROJECT_NAME} PRIVATE CI_BUILD_NUMBER=\"${CI_BUILD_NUMBER}\")
+endif()
```

Comment:
better to use the definition for particular source files. Otherwise, ccache on CI will work worse